blog/2021-08-14-react-navigation-6.0.md

```diff
@@ -0,0 +1,159 @@
+---
+title: React Navigation 6.0
+author: Satyajit Sahoo
+author_url: https://twitter.com/satya164
+author_title: Core Team
+author_image_url: https://avatars2.githubusercontent.com/u/1174278?s=200&v=4
+tags: [release, announcement]
+---
+
+The documentation is now live at [reactnavigation.org](https://reactnavigation.org), and v5 lives [here](/docs/5.x/getting-started).
+
+Largely, React Navigation 6 keeps the same core API as React Navigation 5, and you can think of it as further polishing what was in React Navigation 5. Let's talk about the highlights of this release in this blog post.
+
+<!--truncate-->
+
+## Highlights
+
+### More flexible navigators
+
+Many of the navigators accept some of the customization options as props, which means, we can’t customize them based on the active screen. To make this possible, we needed to move these props to options that you can configure per screen.
+
+In React Navigation 6, many of the props are now screen's options, the most significant ones are `tabBarOptions` and `drawerContentOptions`, where all of them are moved to `options` instead. For example,
+
+Before:
+
+```js
+<Tab.Navigator
+  tabBarOptions={{
+    inactiveTintColor: 'rgba(255, 255, 255, 0.5)',
+    activeTintColor: '#fff',
+    style: {
+      position: 'absolute',
+      borderTopColor: 'rgba(0, 0, 0, .2)',
+    },
+  }}
+>
+```
+
+After:
+
+```js
+<Tab.Navigator
+  screenOptions={{
+    tabBarInactiveTintColor: 'rgba(255, 255, 255, 0.5)',
+    tabBarActiveTintColor: '#fff',
+    tabBarStyle: {
+      position: 'absolute',
+      borderTopColor: 'rgba(0, 0, 0, .2)',
+    },
+  }}
+>
+```
+
+See [deprecations](/docs/upgrading-from-5.x#deprecations) for more details.
+
+### Elements Library
+
+We also took out some of the components and helpers used across various navigators in React Navigations and published them under a new package called [`@react-navigation/elements`](/docs/elements). It can be useful if you're building your own navigator, or just want to reuse some of the components in your app.
+
+Right now we have a small set of components, but there’s more to come.
+
+### Simpler APIs for existing functionality
+
+We made many APIs simpler with React Navigation 6 to address common use cases. For example:
```

Comment:
```suggestion
We simplified many APIs with React Navigation 6 to address common use cases. For example:
```

made simpler -> simplified

usually better to use verb form of the word than "made ${adjective}"